rules/linux/execution_process_started_in_shared_memory_directory.toml

```diff
@@ -32,13 +32,13 @@ references = [
 risk_score = 73
 rule_id = "3f3f9fe2-d095-11ec-95dc-f661ea17fbce"
 severity = "high"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "BPFDoor"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "BPFDoor", "Elastic Endgame"]
 timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
 process where event.type == "start" and
-    event.action == "exec" and user.name == "root" and
+    event.action : ("exec", "exec_event") and user.name == "root" and
```

Comment:
So later when we remove these endgame references, how will we remember that `exec_event` is for endgame vs `exec`?

Reply:
We will need to do the same evaluation process, but we could easily refer to these PRs to see what was explicitly added for Endgame compatibility.